<ui version="4.0" >
 <author>Dane Springmeyer</author>
 <class>Ecotourism</class>
 <widget class="QDialog" name="Ecotourism" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OpenOceanMap - Interview</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QGroupBox" name="vessel_info" >
       <property name="font" >
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title" >
        <string>Ecotourism Info</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" native="1" name="home_port_5" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>4</number>
           </property>
           <property name="leftMargin" >
            <number>4</number>
           </property>
           <property name="topMargin" >
            <number>4</number>
           </property>
           <property name="rightMargin" >
            <number>4</number>
           </property>
           <property name="bottomMargin" >
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="employee_text" >
             <property name="minimumSize" >
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>Type of Empolyee:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="employee_comboBox" >
             <property name="maximumSize" >
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text" >
               <string/>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Owner</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Captain of Boat</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Other</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" native="1" name="vessel_length" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>4</number>
           </property>
           <property name="leftMargin" >
            <number>4</number>
           </property>
           <property name="topMargin" >
            <number>4</number>
           </property>
           <property name="rightMargin" >
            <number>4</number>
           </property>
           <property name="bottomMargin" >
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="vessel_length_text" >
             <property name="minimumSize" >
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>Vessel length (meters) :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="vessel_length_line" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="minimumSize" >
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="contextMenuPolicy" >
              <enum>Qt::PreventContextMenu</enum>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" native="1" name="vessel_length_2" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>4</number>
           </property>
           <property name="leftMargin" >
            <number>4</number>
           </property>
           <property name="topMargin" >
            <number>4</number>
           </property>
           <property name="rightMargin" >
            <number>4</number>
           </property>
           <property name="bottomMargin" >
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="vessel_motor_text" >
             <property name="minimumSize" >
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>Vessel motor (horsepower):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="vessel_motor_line" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="minimumSize" >
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" native="1" name="vessel_length_3" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>4</number>
           </property>
           <property name="leftMargin" >
            <number>4</number>
           </property>
           <property name="topMargin" >
            <number>4</number>
           </property>
           <property name="rightMargin" >
            <number>4</number>
           </property>
           <property name="bottomMargin" >
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="haul_capacity_text" >
             <property name="minimumSize" >
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>Haul Capacity (kilograms):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="haul_capacity_line" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="minimumSize" >
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" native="1" name="home_port" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>4</number>
           </property>
           <property name="leftMargin" >
            <number>4</number>
           </property>
           <property name="topMargin" >
            <number>4</number>
           </property>
           <property name="rightMargin" >
            <number>4</number>
           </property>
           <property name="bottomMargin" >
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="home_port_text" >
             <property name="minimumSize" >
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>Home Port:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="home_port_line" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="minimumSize" >
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" native="1" name="home_port_4" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QLabel" name="workers_text" >
             <property name="minimumSize" >
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>350</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>How many work for the company?</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="workers_line" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="minimumSize" >
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" native="1" name="button_box" >
     <layout class="QHBoxLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbnCancel" >
        <property name="font" >
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbnSelectEcotourism" >
        <property name="font" >
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text" >
         <string>Select Ecotourism Type</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
